app/assets/stylesheets/application.scss

```diff
@@ -1,13 +1,3 @@
-@import 'jquery.ui.theme';
-@import 'jquery.ui.core';
-@import 'jquery.ui.menu';
-//we need to import all the above because we are using jquery-ui-rails and it uses sprockets require instaed of sass import.
-@import 'jquery.ui.autocomplete';
-@import 'jquery.ui.spinner';
-@import 'jquery.ui.spinner_custom';
+@import 'jquery.gridster';
```

Comment:
arent you loading it twice? https://github.com/theforeman/foreman/pull/5212/files#diff-e2cda3aede73b114b7655fdf896b7385R12 ?

Reply:
Didn't we load it on all pages before?
https://github.com/theforeman/foreman/pull/5212/files#diff-f859e2848b07324f808ce1a5ccd9fcbaL13
https://github.com/theforeman/foreman/pull/5212/files#diff-e9457d01ba037a2e0005c43d84bd656eL1

Reply:
OK, thanks for the clarification.